.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     - name: yarn test-viewer
       run: xvfb-run --auto-servernum yarn test-viewer
     - name: yarn test-treemap
-      run: xvfb-run --auto-servernum yarn test-treemap
+      run: xvfb-run --auto-servernum yarn test-treemap || yarn test-treemap
```

Comment:
this wouldn't run it with xvfb the second time.

Reply:
> this wouldn't run it with xvfb the second time.

false.


![image](https://user-images.githubusercontent.com/39191/106546635-83a41880-64c0-11eb-8324-8279212061e3.png)

Reply:
But it's two commands? Are these not processed like how a shell would process commands?

Reply:
> But it's two commands? Are these not processed like how a shell would process commands?

TBF, i had the same thought. i didn't really expect it to work like this. i tried other syntax first but then tested things out and this was the one that worked.  
why? i dunno.

bash is so full of magic.

Reply:
I believe this is the `xvfb-run` entrypoint is written. 

Essentially it sees: `xvfb-run --flags "COMMAND"`, so what happens here is that the command is actually `yarn test-treemap || yarn test-treemap` (run one, then the next, then exit). 

So xvfb is running both within the environment then shutting down.

Reply:
The assumption I made was this entire line would be run as `bash -c '...'` which would internet the OR as a shell normally does. xvfb-run would tear down and then bash would run the right part of the OR, without xvfb-run.

I guess the yml workflow steps don't run commands like that. More like exec in node I suppose, which I think has same behavior...